en/about.html

```diff
@@ -44,7 +45,91 @@
+                <span id="this-span" class="bg-midnight opacity-70 h-10 absolute bottom-0 left-0 z-0"></span>
+            </div>
+            <img class="shadow-2xl rounded-tr-md" src="../assets/joel-muniz-A4Ax1ApccfA-unsplash 2.png" alt="An assortment of vegetables">
+        </div>
+        <div class="class='parent'">
+            <div class="p-10 child inline-block-child">  
+                <!--Card 0-->
+                <div class="card-width rounded overflow-hidden shadow-lg">
+                  <p class="mb-10" id="p-tag-about">BE THE CHANGE</p>
+                  <div class="px-6 py-4">
+                    <p class="text-gray-700 text-base mb-20">
+                        our drivers pick-up excess healthy foods from local food donors and deliver to local food banks and organizations in need. 
+                    </p>
+                  </div>
+                </div>
+              </div>
+            <div class="p-10 child inline-block-child">  
+                <!--Card 1-->
+                <div class=" card-width rounded overflow-hidden shadow-lg">
+                  <img class="w-full" src="../assets/ismael-paramo-tnVdQGmWtb0-unsplash 1.png" alt="Mountain">
+                  <div class="px-6 py-4">
+                    <div class="font-bold text-xl mb-2">DELIVER</div>
+                    <p class="text-gray-700 text-base">
+                        Become a driver: sign up with us and become a part of the mission. View the app to see available pick-ups and drop-offs that fit you schedule.
+                    </p>
+                  </div>
+                </div>
+              </div>
+              <div class="p-10 child inline-block-child">  
+                <!--Card 1-->
+                <div class=" card-width rounded overflow-hidden shadow-lg">
+                  <img class="w-full" src="../assets/jonah-pettrich-nBdUPd4EpB4-unsplash 1.png" alt="Mountain">
+                  <div class="px-6 py-4">
+                    <div class="font-bold text-xl mb-2">DONATE</div>
+                    <p class="text-gray-700 text-base">
+                        Donate your leftover food: register your food availability, have your food picked up by our reliable delivery drivers and save on disposal costs!
+                    </p>
+                  </div>
+                </div>
+              </div>
+              <div class="p-10 child inline-block-child">  
+                <!--Card 4-->
+                <div class="card-width rounded overflow-hidden shadow-lg">
+                  <img class="w-full" src="../assets/melanie-lim-nD3wunG16Fo-unsplash 1.png" alt="Mountain">
+                  <div class="px-6 py-4">
+                    <div class="font-bold text-xl mb-2">RECIEVE</div>
+                    <p class="text-gray-700 text-base">
+                        recieve food donations: communicate your food needs and details for delivery, purchase nutritious food at minimal cost to provide to your community
```

Comment:
Same typo again, gonna look across all files sometime before or after merging
```suggestion
                        receive food donations: communicate your food needs and details for delivery, purchase nutritious food at minimal cost to provide to your community
```